core/src/com/unciv/ui/popups/options/MultiplayerTab.kt

```diff
@@ -117,10 +119,33 @@ private fun addMultiplayerServerOptions(
     serverIpTable.add("Server address".toLabel().onClick {
         multiplayerServerTextField.text = Gdx.app.clipboard.contents
         }).colspan(2).padBottom(Constants.defaultFontSize / 2.0f).row()
+
+    val errorTextField = "".toLabel().also {
+        it.color = Color.RED
```

Comment:
Setting Actor.color on a Label is different from using Label.style.fontColor - which you can simply pass as first parameter to toLabel(). Is this tested?